runtime/default/php.ini

```diff
@@ -0,0 +1,25 @@
+extension_dir=/opt/php/extensions
+display_errors=On
```

Comment:
will those be rendered to the end-user like in regular php?

Reply:
No this is for the default runtime which doesn't run over PHP-FPM or similar. Any error in there will end up in Cloudwatch.

I need to test this out more to make it perfect, this is part of #87